### PR TITLE
Reverts metainfo license to CC0-1.0

### PR DESCRIPTION
**Pull request by A6GibKm**
Closes: #3277. 

**cc**: @stephanlachnit  is this Ok?

### CODE REVIEW

File: debian/copyright [hunk 1]
```diff
@@ -8,3 +8,7 @@ Copyright: 2009-2020 Mathieu Comandon <strider@strycore.com>
 License: GPL-3.0-or-later
  On Debian systems, the complete text of the General Public License version 3
  can be found in "/usr/share/common-licenses/GPL-3".
+
+Files: share/metainfo/net.lutris.Lutris.metainfo.xml
+Copyright: Lutris Team <strider@lutris.net>
+License: CC0-1.0
```
Comment (stephanlachnit) [inline debian/copyright:14]: Add to line 14
```
License: CC0-1.0
 On Debian systems, the complete text of the CC0-1.0 license
 can be found in "/usr/share/common-licenses/CC0-1.0".
```

**Review by stephanlachnit — CHANGES_REQUESTED** [review]

**Comment by stephanlachnit** [comment]
Btw, the GPL v3 is a perfectly _valid_ license for the metainfo. It just happens that some tools might want to generate a catalog from all apps with metainfo in a certain repository, let's say Flathub or the Debian repository.
It's simply easier to distribute this catalog if every metainfo is licensed under a public domain license, since you don't need to care about copyleft and pedantic attribution.

**Comment by A6GibKm** [comment]
Then I wonder why the tool made by appstream, intended to enforce their own spec fails on this particular license for metainfo. I asked in places, and the response was: it does not make sense to use a code license for an asset.

**Comment by stephanlachnit** [comment]
Explaination from the documentation:

> Omitting the license value can result 			in your data not being incorporated into the distribution metadata (so this is a required tag). 		
> A permissive license ensures your data can be combined with arbitrary other data in one file (this means copyleft licenses like the GPL are not suitable as metadata license).

So yeah it's mainly to simplify distribution. Maybe valid is the wrong word here, the metadata might not be shown in a distribution if it's not permissive, thus the warning. At least for KDE Discover, I was able to find it on a local install.

**Review by stephanlachnit — APPROVED** [review]

**Review by strycore — APPROVED** [review]

**Pull request merged**